judgments/forms/search_forms.py

```diff
@@ -35,6 +35,24 @@ def _get_choices_by_group(courts):
 TRIBUNAL_CHOICES = _get_choices_by_group(all_courts.get_grouped_selectable_tribunals())
 
 
+def get_short_identifiers(top_level_choices):
+    all_shorts = set()
+    for key, value in top_level_choices:
+        if isinstance(value, list):
+            all_shorts = all_shorts | set(x[0].partition("/")[0] for x in value)
+    return all_shorts
```

Comment:
We could move this under the `CourtTribunalField` class as it's only being used there.

Reply:
I saw this as being directly analogous to "_get_choices_by_group" (and probably should have had an underscore); it doesn't operate on a CourtTribunalField so I'm not sure it belongs there.